### ISSUE TITLE
Android App Sunsetting - Post-Teardown Work

### ISSUE THREAD

**Issue by JN-Hernandez** [azavea/cac-tripplanner-android#224]
The GoPhillyGo Android app has been removed from the Google Play store and is being sunset ([Outline Overview of full process here](https://github.com/azavea/cac-tripplanner/issues/1366)).  This issue is to track the last remaining work:

- [x] Archiving the GitHub Repo
- [x] Removing notifications
    - [x] Slack notifications
    - [x] Firebase crash reporting
    - [x] Rollbar notifications

Investigation performed during the outline overview process determined that there are no AWS resources to reclaim from sunsetting this app, nor is there any stored user/application data to be backed up.

**Comment by JN-Hernandez** [comment]
**Slack Notifications**
The slack notifications for the GoPhillyGo app was set up through email forwarding, example [Slack Notification](https://element84.slack.com/archives/C04UC67U8SJ/p1701372495008839):
![Screenshot 2024-02-06 at 14 22 33](https://github.com/azavea/cac-tripplanner-android/assets/66323217/12f610ab-ae06-4216-afbb-6adcea17c10e)

Confirmed that no other slack notifications have been set up for the application.  The `@azavea.com` domain has been closed and will no longer receive/forward emails to the slack channel, effectively removing notifications.

**Firebase Crash Reporting**
The GoPhillyGo android app had Firebase Crash Reporting set up, which is a Google product.  The `@azavea.com` domain no longer exists, and attempts to log into the `azaveadev@azavea.com` Firebase account will fail.  I've emailed the [Firebase Support Team](https://support.google.com/firebase/contact/support) to confirm if the project has been removed or any additional steps need to be taken.

**Rollbar**
Confirmed that there isn't any Rollbar set-up for the GoPhillyGo Android app - confirmed with @rachelekm as different users have different views into Rollbar.
